poker-pals/src/components/UserNavbar.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+    <nav>
+        <b-navbar toggleable="sm">
+            <b-navbar-brand href="#" class="mx-3">Poker Pals</b-navbar-brand>
+            <b-navbar-toggle target="nav-collapse"></b-navbar-toggle>
+            <b-collapse id="nav-collapse" is-nav>
+                <!-- Right aligned nav items -->
+                <b-navbar-nav class="ml-auto mx-3">
+                    <b-nav-item to="profile" class="mx-2">
+                        <img class="mr-2 img-fluid" src="../assets/chip.png" alt="Poker Chip"/>
+                        <span class="b-nav-text">1,200,450</span>
+                    </b-nav-item>
+                    <b-nav-item to="profile">
+                        <img class="img-fluid" src="../assets/player_icon_1.png" alt="User Profile Image"/>
+                    </b-nav-item>
+                </b-navbar-nav>
+            </b-collapse>
+        </b-navbar>
+    </nav>
+</template>
+
+<style scoped>
+    @import url('https://fonts.googleapis.com/css?family=Pacifico&display=swap');
```

Comment:
Would be good to move this to `App.vue`. I changed it on your login branch too.

Reply:
Fixed this, moved the icons import too.